src/Microsoft.Data/PrimitiveColumnContainer.cs

```diff
@@ -25,6 +31,7 @@ public PrimitiveColumnContainer(T[] values)
             {
                 curBuffer = new DataFrameBuffer<T>();
                 Buffers.Add(curBuffer);
+                NullBitMapBuffers.Add(new DataFrameBuffer<byte>());
```

Comment:
Is it possible to lazy initialize the NullBitMapBuffers? For example, if I don't have any null values, there should be no needed to create these objects.

Reply:
I did consider lazy initialization, but the C# Arrow implementation seems to always maintain a null bit map, so I went with this.